.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,35 @@
+# workflow name
+name: Unit Tests
+
+# execute on events
+on:
```

Comment:
In which scenarios will we run it?

Reply:
Configuration suggests that the tests will run automatically when there is a push event on the main branch